### DISCUSSION TITLE
S-HOOK 레벨4 슬기롭게 헤쳐나가기

### DISCUSSION THREAD

**Discussion by giscus** [Cyma-s/Cyma-s.github.io#78] [General]
# S-HOOK 레벨4 슬기롭게 헤쳐나가기

LOG 알고리즘 3문제 풀었다. 리뷰 달고 리뷰 반영하느라 40분 밖에 투자 못했는데 MST를 구하는 코드가 그렇게 어렵지 않아서 그 사이 3문제나 풀었따! 근데 사실 다 이전 문제 코드를 재활용하는 느낌이라 사실상 한 문제를 푼 거나 다름없다...…

https://vero.wiki/shook-level4/

**Comment by hong-sile** [comment]
팀에서 고민하고 꽤 관심 있는 주제인데 여기서 명시적으로 정하고 시도하는군요.

나중에 팀이 어떻게 진행되었는지 구경하러 가겠습니다.

**Reply by Cyma-s** [reply]
환영입니다 저는 이야기 해보는 건 좋다고 생각해요!
